### PR TITLE
perf: add rdp connection speed option

### PR DESCRIPTION
**Pull request by fit2bot**
perf: add rdp connection speed option

### CODE REVIEW

File: apps/authentication/api/connection_token.py [hunk 1]
```diff
@@ -116,6 +146,9 @@ def get_rdp_file_info(self, token: ConnectionToken):
         rdp = token.asset.platform.protocols.filter(name='rdp').first()
         if rdp and rdp.setting.get('console'):
             rdp_options['administrative session:i'] = '1'
+        print(token.connect_options)
+        rdp_connection_speed = token.connect_options.get('rdp_connection_speed', 'auto')
+        rdp_options.update(RDP_CONNECTION_SPEED_OPTION_MAP.get(rdp_connection_speed, {}))
 
         # 文件名
         name = token.asset.name
```
Comment (BaiJiangJie) [inline apps/authentication/api/connection_token.py:154]: This code snippet is quite extensive and includes various variables, functions, and queries to determine the configuration of a Remote Desktop Protocol (RDP) connection through Azure AD B2C.

Here's an overview on what I consider:

**Potential Issues**: 

- The `print` statement at line 175 isn't directly related to the core functionality being tested. It may serve as a debugging tool but it could be removed without changing anything significantly.

**Insufficient Documentation**: 
- While the code looks mostly correct, there seems to be some missing context that would give more information about the environment or purpose of these configurations. For instance, why do they use RDP connections? Does their system run on Windows Server?

**Optimization Suggestions:**

1. **Variable Usage**: Simplify variable names. This can make reading the code easier if other developers will read or maintain this code. Replace:
   - "token" with "azure_token"
   
   - "name" with "asset_name"

2. **Function Naming**: Ensure function names are readable and descriptive. Function "get_rdp_file_info" doesn't seem relevant due to its complexity; you might want something like "generate_rdp_config":

```
def generate_rdp_configuration(azure_token: Optional[ConnectionToken] = None)
```

3. **Coding Style**: Use consistent indentation which makes readability better (`tab` instead of one space).

4. **Documentation**: Implement proper commentaries around important points, such as when using different connection speeds. Add comments around complex logic sections (e.g., `if elif else structure`) making it clearer how conditions are evaluated.

5. **Testing**: Include unit tests that cover all corner cases including edge scenarios in case the code handles specific input gracefully. Consider adding assertions for output parameters.

File: apps/authentication/api/connection_token.py [hunk 2]
```diff
@@ -116,6 +146,8 @@ def get_rdp_file_info(self, token: ConnectionToken):
         rdp = token.asset.platform.protocols.filter(name='rdp').first()
         if rdp and rdp.setting.get('console'):
             rdp_options['administrative session:i'] = '1'
+        rdp_connection_speed = token.connect_options.get('rdp_connection_speed', 'auto')
+        rdp_options.update(RDP_CONNECTION_SPEED_OPTION_MAP.get(rdp_connection_speed, {}))
 
         # 文件名
         name = token.asset.name
```
Comment (BaiJiangJie) [inline apps/authentication/api/connection_token.py:153]: The main issue I found with this code snippet is that it contains several typos and missing or extra spaces which affect readability.

Here are some suggested changes:

1. Change the comment above Line 7 to English (it's currently German).
2. Replace `@@@` with an actual newline separator between different sections of the code.
3. Move all variable assignments to their own lines for improved clarity.
4. Correct indentation errors, especially at line 5, where it should be indented appropriately.
5. Add comments to explain what each section does; unnecessary blank lines can often clutter the documentation text block.

Some parts also look like they have typo in them as well:

- It should be "is_true" instead of just "bool".
- In "RDP_CONNECTION_SPEED_OPTIONS", there might not be a space after "R". Also, remove trailing spaces around `option`.

Let me know if you want these modifications applied or if you need anything else.

**Review by LeeEirc — APPROVED** [review]

**Review by BaiJiangJie — APPROVED** [review]
